core/templates/services/schema-submitted.service.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service provides emitters for events based on
+ * submission of different schema forms
+ */
+
+import { Injectable, EventEmitter } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SchemaSubmittedService {
+  private _submittedSchemaBasedFormEventEmitter = new EventEmitter();
+
+  get onSubmittedSchemaBasedForm() {
+    return this._submittedSchemaBasedFormEventEmitter;
+  }
+}
+
+angular.module('oppia').factory(
+  'SchemaSubmittedService', downgradeInjectable(SchemaSubmittedService));
```

Comment:
A bit dubious about the name for this file... Can you explain why you think this is an apt name?

Reply:
This just houses the event emitter corresponding to submitting schema forms. What would you suggest I change it to?

Reply:
@nithusha21 ?

Reply:
Sorry for the delay. I was confused because you don't submit a schema right? I am not sure. Maybe I have got it wrong not sure...

My suggestion would be Schema-form-submitted probably

Reply:
Right, I'll refactor it